bokeh/models/sources.py

```diff
@@ -58,6 +59,11 @@ class ColumnDataSource(ColumnarDataSource):
 
           source = ColumnDataSource(data)
 
+    .. note::
+        ``ColumnDataSource`` creates a shallow copy of the passed ``dict``. Use
```

Comment:
Sphinx is very finicky, this will fail the docs build, it just needs a blank line after the `.. note::`

Reply:
At the bottom of the very same docstring there is:
```
.. note::
        There is an implicit assumption that all the columns in a given
        ``ColumnDataSource`` all have the same length at all times. For this
        reason, it is usually preferable to update the ``.data`` property
        of a data source "all at once".
```
So I'm assuming it would work, but I will make the change as sphinx' documentation clearly states that a blank line after the directive is the intended use.

Reply:
Oh, that's a mistake then. Can you fix that one as well, as long as we are editing this docstring?

Reply:
Perhaps this has changed, though? I know for certain it used to be required but it seems to be rendering correctly, and it's not causing any warnings (which are considered errors). Still, I'd prefer if they were made consistent with everywhere else in the codebase.